barretenberg/cpp/src/barretenberg/ecc/curves/bn254/pairing_impl.hpp

```diff
@@ -287,18 +295,32 @@ fq12 reduced_ate_pairing_batch(const g1::affine_element* P_affines,
                                const g2::affine_element* Q_affines,
                                const size_t num_points)
 {
+
     std::vector<g1::element> P(num_points);
```

Comment:
nit but it would be more natural here to not initialize these then just push back. Also, a comment or two wouldn't hurt here